interfaces/mount/entry.go

```diff
@@ -103,3 +108,36 @@ func (e Entry) String() string {
 	return fmt.Sprintf("%s %s %s %s %d %d",
 		name, dir, fsType, options, e.DumpFrequency, e.CheckPassNumber)
 }
+
+// ParseEntry parses a fstab-like entry.
+func ParseEntry(s string) (Entry, error) {
+	var e Entry
+	var err error
+	var df, cpn int
+	fields := strings.FieldsFunc(s, func(r rune) bool { return r == ' ' || r == '\t' })
+	// do all error checks before any assignments to `e'
+	if len(fields) < 4 || len(fields) > 6 {
+		return e, fmt.Errorf("expected between 4 and 6 fields, found %d", len(fields))
+	}
+	// Parse DumpFrequency if we have at least 5 fields
+	if len(fields) >= 5 {
+		df, err = strconv.Atoi(fields[4])
+		if err != nil {
+			return e, fmt.Errorf("cannot parse dump frequency: %s", err)
```

Comment:
In these cases, rather than printing the error it's much more useful to have the original string quoted:

`return e, fmt.Errorf("cannot parse dump frequency: %q", fields[4])`